src/dxvk/imgui/dxvk_imgui.cpp

```diff
@@ -1748,13 +1800,16 @@ namespace dxvk {
       ImGui::Checkbox("Scene Left-Handed", &RtxOptions::Get()->isLHSObject());
       fusedWorldViewModeCombo.getKey(&RtxOptions::Get()->fusedWorldViewModeRef());
       ImGui::Separator();
-
```

Comment:
Revert the empty line changes. Generally should avoid making any unnecessary/unwanted changes as it complicates the PR.